src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -458,4 +436,56 @@
   <Target Name="CreateRuntimeRootIlLinkDescFile" BeforeTargets="CoreCompile" DependsOnTargets="_CreateILLinkRuntimeRootDescriptorFile"/>
 
   <Import Project="$(RepositoryEngineeringDir)codeOptimization.targets" />
+
+  <!-- Setup eventing file generation -->
+  <Target Name="FindPythonWindows"
```

Comment:
It feels a bit off to add a bunch of targets to the CoreCLR  build to discover the python executable.

Did you consider putting them in a more central location and then using a property, like `<RequiresPython>true</RequiresPython>` to allow projects to opt into using it? That would make it more generally useful and clean up this project file.

Reply:
I'll go move them up so more projects can opt-in.

Reply:
Now any target can depend on the `FindPython` target and python will be available for their scripts to use.